microovn/node/node.go

```diff
@@ -0,0 +1,2 @@
+// Package node provides functions operating on nodes in the cluster.
```

Comment:
When adding package level Godoc for the `node` package, I have to admit the name appears a bit vague, having our discussion this morning in mind cc @mkalcok @crypticC0der